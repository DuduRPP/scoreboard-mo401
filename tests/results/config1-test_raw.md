| Instruction/Cycle   |   Issue |   Read |   Execute |   Write |
|:--------------------|--------:|-------:|----------:|--------:|
| fld f2, x1          |       1 |      2 |         3 |       4 |
| fadd f4, f2, f6     |       2 |      5 |         7 |       8 |
| fsub f8, f4, f2     |       9 |     10 |        12 |      13 |
