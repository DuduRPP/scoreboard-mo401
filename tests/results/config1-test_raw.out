| Instruction     |   Issue |   Read |   Execute |   Write |
|:----------------|--------:|-------:|----------:|--------:|
| fld f2, x1      |       1 |      2 |         3 |       4 |
| fadd f4, f2, f6 |       2 |      4 |         6 |       7 |
| fsub f8, f4, f2 |       8 |      9 |        11 |      12 |
